weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallPage.tsx

```diff
@@ -1,18 +1,23 @@
 import {Box} from '@material-ui/core';
 import React, {FC} from 'react';
 import {useHistory} from 'react-router-dom';
+import {Loader} from 'semantic-ui-react';
```

Comment:
We should probably have design finalize a common component for loader indication so we can start replacing this everywhere, but for now this aligns with the main app.